app/graph/types/team_type.rb

```diff
@@ -301,6 +301,6 @@ def shared_teams
   end
 
   def tipline_messages(uid:)
-    object.tipline_messages.where(uid: uid).last(100)
```

Comment:
do you still need a limit on this query?

Reply:
No and I removed the limit and sort items by id `DESC`